tests/Browser/BlogManageTest.php

```diff
@@ -0,0 +1,562 @@
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->type('title', 'Lốc bụi cao 650 m trên sao Hỏa')
+                    ->script("CKEDITOR.instances['blog-content'].setData('content');");
+                    
+            $browser->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('Create new blog success !')
+                    ->assertSee('Lốc bụi cao 650 m trên sao Hỏa');
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.2: Create blog: Fill all input but fill in title less than 5 characters
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogFillTitleLessThan5Character()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->type('title', 'Lốc')
+                    ->script("CKEDITOR.instances['blog-content'].setData('content');");
+
+            $browser->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The title must be at least 5 characters.')
+                    ->assertDontSee('Lốc bụi');
+        });
+    }
+
+    /**
+     * Test create blog with user can see blog.
+     * case 2.3: Create blog: Fill all input but fill in title over 100 characters
+     * Error
+     *
+     * @return void
+     */
+    public function testCreateBlogFillTitleOver100Character()
+    {
+        $user = $this->userCanSee();
+        $data = $this->getDataTest();
+        $this->browse(function ($browser) use ($user, $data) {
+            //string over 100 character
+            $textOver100 = str_repeat('a', 101);
+
+            $browser->visit($data['path']['login'])
+                    ->loginAs($user)
+                    ->visit($data['path']['blog'])
+                    ->press('Create Blog')
+                    ->waitFor('#createModal')
+                    ->type('title', $textOver100)
+                    ->script("CKEDITOR.instances['blog-content'].setData('content');");
+
+            $browser->waitForText('Submit')
+                    ->press('Submit')
+                    ->waitForLocation($data['path']['blog'])
+                    ->assertSee('The title may not be greater than 100 characters.')
+                    ->assertDontSee($textOver100);
```

Comment:
When does blog list display 'content' so you can assert ?

Reply:
I fixed. I change variable $textOver100 to $textTitleOver100